server/svix-server/src/worker.rs

```diff
@@ -298,10 +311,10 @@ pub async fn worker_loop(
                         tokio::spawn(async move {
                             if let Err(err) = dispatch(cfg, &pool, cache, &queue_tx, msg).await {
                                 tracing::error!("Error executing task: {}", err);
-                                queue_tx.nack(delivery).await.unwrap();
+                                queue_tx.nack(delivery).await.expect("Error executing task");
```

Comment:
This is not what this error means, it's an error contact redis when trying to nack (needs better wording).